server/src/session.ts

```diff
@@ -708,8 +708,12 @@ export class Session {
       this.logger.info(`Failed to get language service for closed project ${project.projectName}.`);
       return undefined;
     }
+    const languageService = project.getLanguageService();
+    if (!('getTcb' in languageService)) {
+      return undefined;
+    }
```

Comment:
nit: Looks like the ts infer `languageService` to `never`, why not use type guards?
```ts
  function isNgLs(ls: ts.LanguageService): ls is NgLanguageService {
    return 'getTcb' in ls;
  }
```

Reply:
Good suggestion, thanks!